stile/sys_tests.py

```diff
@@ -1157,17 +1338,15 @@ class ScatterPlotStarVsPSFG1SysTest(ScatterPlotSysTest):
     required_quantities = [('g1', 'g1_err', 'psf_g1')]
 
     def __call__(self, array, per_ccd_stat = 'None', color = '', lim=None):
-        per_ccd_stat = None if per_ccd_stat == 'None' else per_ccd_stat
-        if per_ccd_stat:
-            psf_g1, g1, g1_err = self.getStatisticsPerCCD(array['CCD'], array['psf_g1'],
-                                                          array['g1'], yerr = array['g1_err'],
-                                                          stat = per_ccd_stat)
-        else:
-            psf_g1, g1, g1_err = array['psf_g1'], array['g1'], array['g1_err']
-        return self.scatterPlot(psf_g1, g1, yerr=g1_err,
-                                xlabel=r'$g^{\rm PSF}_1$', ylabel=r'$g^{\rm star}_1$',
-                                color=color, lim=lim, equal_axis=False,
-                                linear_regression=True, reference_line='one-to-one')
+        self.per_ccd_stat = None if per_ccd_stat == 'None' else per_ccd_stat
```

Comment:
This is because when we pass `None` through the command option in HSC pipeline, it's treated as string. However, I think this line should be put into `hsc/sys_test_adapters.py` as it's related to the pipeline. I've done this change.